readthedocs build failing on main branch

See e.g., https://readthedocs.org/projects/circuitpython/builds/11523378/ (may need to be logged in to RTD as a CircuitPython collaborator)

```
Running Sphinx v1.8.5
Traceback (most recent call last):
  File "tools/extract_pyi.py", line 15, in <module>
    import isort
ModuleNotFoundError: No module named 'isort'
make: *** [stubs] Error 1

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/circuitpython/envs/latest/lib/python3.7/site-packages/sphinx/config.py", line 368, in eval_config_file
    execfile_(filename, namespace)
  File "/home/docs/checkouts/readthedocs.org/user_builds/circuitpython/envs/latest/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/circuitpython/checkouts/latest/conf.py", line 77, in <module>
    subprocess.check_output(["make", "stubs"])
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/subprocess.py", line 395, in check_output
    **kwargs).stdout
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/subprocess.py", line 487, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['make', 'stubs']' returned non-zero exit status 2.
```

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -3,3 +3,4 @@ recommonmark==0.6.0
 sphinxcontrib-svg2pdfconverter==0.1.0
 astroid
 sphinx-autoapi
+isort

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -4,3 +4,4 @@ sphinxcontrib-svg2pdfconverter==0.1.0
 astroid
 sphinx-autoapi
 isort
+black

